abb_irb1200_support/package.xml

```diff
@@ -36,11 +36,11 @@
 
   <build_depend>roslaunch</build_depend>
```

Comment:
<build_depend>roslaunch</build_depend> should be replaced with  <depend>roslaunch</depend>

Reply:
replaced with `<test_depend>roslaunch</test_depend>`, as suggested by @gavanderhoorn